src/Core/Domain/Carrier/Command/EditCarrierCommand.php

```diff
@@ -143,4 +150,64 @@ public function setLogoPathName(?string $logoPathName): self
 
         return $this;
     }
+
+    public function hasAdditionalHandlingFee(): ?bool
+    {
+        return $this->hasAdditionalHandlingFee ?? null;
+    }
+
+    public function setAdditionalHandlingFeeStatus(bool $hasAdditionalHandlingFee): self
```

Comment:
```suggestion
    public function setAdditionalHandlingFee(bool $hasAdditionalHandlingFee): self
```
It's preferable if getters and setters use the same naming